examples/jsm/nodes/display/NormalMapNode.js

```diff
@@ -57,7 +57,7 @@ class NormalMapNode extends TempNode {
 		if ( scaleNode !== null ) {
 
 			const normalMapScale = mul( normalMap.xy, scaleNode );
-			normalMap = join( normalMapScale, normalMap.z );
+			normalMap = vec3( normalMapScale, normalMap.z );
```

Comment:
I wouldn't remove the `join`, this can be useful and it's a modern variable constructor, I think we can rename it to `auto()`?

Reply:
If you didn't remove it from `ShaderNodeElements` you can keep it like this :)

Reply:
I still think it is better to use a specific constructor...

Reply:
For this application I agree but for example, `join` or `auto` can be used for others case like in `JoinEditor`  of `NodeEditor` for a dynamic vector creation.

Reply:
Maybe then it would be better to transfer the JoinEditor functionality to the Vector2Editor, Vector3Editor, so on, using the `vec2`, `vec3` functions?

Reply:
In this case, I think that it is better to add a `select` menu in `JoinEditor` to select the type of output.

Reply:
Should this be another PR or should it be done in this one? I am not very familiar with how NodeEditor works...

Reply:
Another PR is better, don't worry about it now.